core-js/src/main/javascript/cdf-legacy/components/core.js

```diff
@@ -168,7 +168,8 @@ BaseComponent = Base.extend(Backbone.Events).extend({
             arr[val[0]]=val[1];
           });
           jXML = Dashboards.parseXActionResult(myself, Dashboards.urlAction(this.url, arr));
-        } else {
+        } else if(this.path || this.action) {
+          //BISERVER-14940 if path and action are both null, the request will fail, as it does not have enough parameters for the request
```

Comment:
Also, in this case, it happens because the Prompting API already provides the `valuesArray` filled in, and there is no need to fetch the values from a service. However, in some cases (of dependent selects), the array is empty.